From 8ad3f6d62c5f1c0060d4344ef195be5d57453c37 Wed Jul 27 11:11:41 2022
From: Ionel-Catalin Mititelu <ionel-catalin.mititelu@intel.com>
Date: Wed, 27 Jul 2022 11:11:41 +0300
Subject: [PATCH] libvhal: Fix_build_with_gcc_11

Include cstdint library in order to fix the compile issue
found after upgrading on gcc 11

---
 include/libvhal/android_audio_core.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/libvhal/android_audio_core.h b/include/libvhal/android_audio_core.h
index aac7c67..f8c3755 100644
--- a/include/libvhal/android_audio_core.h
+++ b/include/libvhal/android_audio_core.h
@@ -17,6 +17,9 @@
  * limitations under the License.
  *
  */
+
+#include <cstdint>
+
 namespace vhal {
 namespace client {
 namespace audio {
-- 
2.34.1

